requirements/cpu-build.txt

```diff
@@ -4,9 +4,9 @@ packaging>=24.2
 setuptools>=77.0.3,<81.0.0
 setuptools-scm>=8
 --extra-index-url https://download.pytorch.org/whl/cpu
-torch==2.8.0+cpu; platform_machine == "x86_64" or platform_machine == "s390x"
+torch==2.9.0+cpu; platform_machine == "x86_64" or platform_machine == "s390x"
 torch==2.9.0; platform_system == "Darwin"
-torch==2.8.0; platform_machine == "ppc64le" or platform_machine == "aarch64"
+torch==2.9.0; platform_machine == "ppc64le" or platform_machine == "aarch64"
```

Comment:
Could we hold-off until we check that torch 2.9 works smoothly on Arm?
I'm hoping to get that checked today/tomorrow.

If you need that merged now, please remove keep the `2.8.0` torch version for AArch64 and I'll update that later.